### PR TITLE
Replace endpoints with diagrams

### PR DESCRIPTION
**Pull request by eric-schuh**
@bumblefudge @jandrieu I believe this branch is ready for merge, replaced the text endpoints sections with the diagrams for the two use cases we have diagrams for so far. Also it is trivial to change the github.io branch that is displayed for future reference: 

Go to settings, scroll down to the bottom and you can change the branch in a dropdown to view how the changes look and then swap back to main easily after you check.

***

### :boom: Error: 504 Gateway Time-out :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on May 3, 2022, 7:44 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c-ccg%2Fvc-api-use-cases%2F4f5df7ed7e8465aa2204cac1c961790dd9bcb561%2Findex.html%3FisPreview%3Dtrue)

```
<html><body><h1>504 Gateway Time-out</h1>
The server didn't respond in time.
</body></html>

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c-ccg/vc-api-use-cases%237.)._
</details>

### CODE REVIEW

**Review by bumblefudge — APPROVED** [review]

**Comment by TallTed** [comment]
These are PNG, so changes can't be suggested directly, and their accessibility is highly questionable.  They'll probably need to become SVG or other vector format that includes text as text, so screen readers work, among other things.

That aside -- I suggest changing the labels that now read "something - role" to "something (role)" (that is, remove the hyphen and wrap the role in parentheses).  For instance, "Riley - Holder" would become "Riley (Holder)".  (Tangentially, this is the only such that doesn't have a newline between the "something" and the "role" -- and I would suggest making all of them have that newline.)

**Comment by eric-schuh** [comment]
@TallTed thanks for the formatting suggestions. Joe and I are working on a script that will allow us to export these diagrams into the swimlanes format as we both agree that the images are not ideal for this medium where we want discussion to happen. Hopefully we can get a more suggestable/editable format in the next couple of weeks. Will be working on the formatting suggestions in the meantime.

**Comment by jandrieu** [comment]
Our translation tool is close. Eric and I will update this with mermaid diagrams and we can tease out the best formatting with that tool.

File: index.html [hunk 1]
```diff
@@ -201,19 +201,20 @@ <h3>Get Digital Permanent Resident Card</h3>
...
+        with a set of Verifiable Credentials that she stores in her digital wallet. A subset of the 
+        Verifiable Credentials that she has received are digitally signed single-use age tokens that 
+        only assert that she is above the age of 21 and are marked as "used" by the TruAge system 
+        when they are submitted as a part of an age-restricted goods purchase, such as buying a 
+        bottle of wine. Eventually, Riley runs out of single use age tokens in her digital wallet. 
+        The digital wallet keeps track of which tokens have been used and once all tokens have been 
+        consumed, contacts a refresh service endpoint listed in one of the TruAge credentials that 
+        provides new over-age tokens. The digital wallet requests a new set of tokens by hitting the 
+        HTTP API of this VC-refresh service listed in the “refreshService” array and POSTing the 
+        original Verifiable Credential containing the refresh service description. The HTTP API
```
Comment (TallTed) [inline index.html:269]:
```suggestion
        VC API of this VC-refresh service listed in the “refreshService” array and POSTing the 
        original Verifiable Credential containing the refresh service description. The VC API 
```

File: index.html [hunk 2]
```diff
@@ -201,19 +201,20 @@ <h3>Get Digital Permanent Resident Card</h3>
...
+        Verifiable Credentials that she has received are digitally signed single-use age tokens that 
+        only assert that she is above the age of 21 and are marked as "used" by the TruAge system 
+        when they are submitted as a part of an age-restricted goods purchase, such as buying a 
+        bottle of wine. Eventually, Riley runs out of single use age tokens in her digital wallet. 
+        The digital wallet keeps track of which tokens have been used and once all tokens have been 
+        consumed, contacts a refresh service endpoint listed in one of the TruAge credentials that 
+        provides new over-age tokens. The digital wallet requests a new set of tokens by hitting the 
+        HTTP API of this VC-refresh service listed in the “refreshService” array and POSTing the 
+        original Verifiable Credential containing the refresh service description. The HTTP API 
+        ensures that it has received a valid credential and reissues a set of new digitally signed
```
Comment (TallTed) [inline index.html:270]:
```suggestion
        ensures that it has received a valid credential and issues a new set of digitally signed 
```

**Comment by eric-schuh** [comment]
@TallTed mermaid is in (there are a couple of render issues I am raising w/Manu) but mostly working. Next on my list are the label changes (and other minor tweaks) you suggested

**Comment by TallTed** [comment]
@jandrieu -- Did you check whether the [label changes (and other minor tweaks)](https://github.com/w3c-ccg/vc-api-use-cases/pull/7#issuecomment-1116495940) I previously suggested ([1](https://github.com/w3c-ccg/vc-api-use-cases/pull/7#pullrequestreview-943423827), [2](https://github.com/w3c-ccg/vc-api-use-cases/pull/7#pullrequestreview-943424052)), which are only recorded here, had been made (by you or @eric-schuh) prior to merging this PR (which merge henceforth conceals my suggestions, unless carried to new issues/PRs)?

**Pull request merged**